HT-2.py

```diff
@@ -0,0 +1,67 @@
+# Підраховує кількість голосних літер у тексті
+
+
+def count_vowels(text):
```

Comment:
Too complicated. Here two key moments:
1. In task was asked to get sum =f lowercase vowel + uppercase vowel. No need to calculate each capital and lower vowel separately. 

 2. for such operation better to use string operation count   -> poem_text.count('a') etc